src/benchmark/Akka.Cluster.Benchmarks/Sharding/ShardMessageRoutingBenchmarks.cs

```diff
@@ -15,18 +15,20 @@
 using Akka.Routing;
 using BenchmarkDotNet.Attributes;
 using BenchmarkDotNet.Diagnostics.dotTrace;
+using BenchmarkDotNet.Engines;
 using static Akka.Cluster.Benchmarks.Sharding.ShardingHelper;
 
 namespace Akka.Cluster.Benchmarks.Sharding
 {
-    [DotTraceDiagnoser]
+    //[DotTraceDiagnoser]
     [Config(typeof(MonitoringConfig))]
+    [SimpleJob(RunStrategy.Monitoring, launchCount: 10, warmupCount: 10)]
```

Comment:
max 20 iterations of each benchmark variant (and they run 2x when `DotTraceDiagnoser` is enabled)